src/Core/Services/Implementations/EventService.cs

```diff
@@ -213,6 +213,28 @@ public async Task LogOrganizationEventAsync(Organization organization, EventType
             await _eventWriteService.CreateAsync(e);
         }
 
+        public async Task LogSsoUserEventAsync(SsoUser ssoUser, EventType type, DateTime? date)
```

Comment:
Looking at this now, I am not sure how useful this event will actually be since it is not tied to an organization user directly. I think we'll need to re-classify it as an organization user event so that it can show up in the web vault under that organization user.

Reply:
Done